feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -1,14 +1,31 @@
+import { runImporter } from "../importer";
+
 export default class DataAccount {
     constructor() {
-        this.importingResults = [];
+        this.importingReports = [];
         this.importedFileNames = [];
-        this.personalData = { name: { givenName: "", familyName: "" } };
-        this.preferredLanguage = [];
         this.analyses = {};
         this.reports = {};
         this.analysesExecutionResults = [];
     }
 
+    async import({ importers, zipFile, pod }) {
+        for (let [attr, importerClass] of Object.entries(importers)) {
+            let { result, report, importedFileNames } = await runImporter({
+                importerClass,
+                zipFile,
+                pod,
+                //account is kept in here for now so we can support the old importer structure
+                account: this,
+            });
+            this.importingReports.push(report);
+            for (let name of importedFileNames || [])
+                this.addImportedFileName(name);
+            if (result) this[attr] = result;
+        }
+        return this;
```

Comment:
As I indicated in the #1190, creating an empty object that then needs to be populated is weird; but it's even weirder to have a method that returns `this`. If a method does that it's, effectively, a constructor. So, as long as you're doing this restructuring, would it be possible to merge the constructor with import? That way the constructor would, unequivocally, be building the object, as they are wont to do.

Reply:
First thing I tried, but it does not work as import always _has to be async_ and wait while constructors _can never be async._ `await runImporter` is absolutely crucial here right?

Reply:
I guess so, since the event loop will have to take care of inputs and things like that. Not totally sure, though. You can't do a lot of stuff while you're importing.